src/scenario.h

```diff
@@ -302,7 +302,7 @@ typedef struct {
 	char custom_strings[0x8000];
 	uint32 game_ticks_1;
 	rct_ride rides[255];
-	uint16 word_01388698;
+	uint16 word_01388698; // TODO: rename to match RCT2_ADDRESS_SAVED_AGE ???
```

Comment:
saved_view_x should really be a `sint16` but I don't think it will ever go negative so it shouldn't be a problem being used as `uint16`.  Where abouts in the code does saved_age get accessed and used as a ride?